mcs_benchmark_data/cli/commands/create_benchmark_pipeline_command.py

```diff
@@ -41,73 +43,48 @@ def __call__(self, args):
         else:
             data_dir = "DATA_DIR_PATH"
 
-        self.make_benchmark_directories(
-            root_path=root_path, benchmark_name=benchmark_name
-        )
-
-        self.create_files_from_template(
-            root_path=root_path, benchmark_name=benchmark_name, data_dir=data_dir
+        self._make_benchmark_directories(

```

Comment:
`_make_benchmark_directories` will have `benchmark_name` from `self`, since you got `args` in the constructor. That was the motivation for passing args into the constructor, and changes that followed from that.